homeassistant/components/irm_kmi/config_flow.py

```diff
@@ -0,0 +1,147 @@
+"""Config flow to set up IRM KMI integration via the UI."""
+
+import asyncio
+import logging
+
+from irm_kmi_api import IrmKmiApiClient
+import voluptuous as vol
+
+from homeassistant.components.zone import DOMAIN as ZONE_DOMAIN
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult, OptionsFlow
+from homeassistant.const import ATTR_LATITUDE, ATTR_LONGITUDE, CONF_ZONE
+from homeassistant.core import callback
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.selector import (
+    EntitySelector,
+    EntitySelectorConfig,
+    SelectSelector,
+    SelectSelectorConfig,
+    SelectSelectorMode,
+)
+
+from .const import (
+    CONF_LANGUAGE_OVERRIDE,
+    CONF_LANGUAGE_OVERRIDE_OPTIONS,
+    CONFIG_FLOW_VERSION,
+    DOMAIN,
+    OUT_OF_BENELUX,
+    USER_AGENT,
+)
+from .types import IrmKmiConfigEntry
+from .utils import get_config_value
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class IrmKmiConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Configuration flow for the IRM KMI integration."""
+
+    VERSION = CONFIG_FLOW_VERSION
```

Comment:
I would not use a variable for the version imo

Reply:
Constant removed and assigned the literal value directly in commit 895eb40104955a96e82a58580f0416dd2043d1e7